agent/scripts/newrelic.ini.template

```diff
@@ -160,7 +160,7 @@ newrelic.daemon.logfile = "/var/log/newrelic/newrelic-daemon.log"
 ;          rarely need to change this from the default, and usually only under
 ;          the guidance of technical support.
 ;          Must be one of the following values:
-;            always, error, warning, info, debug
+;            always, error, warning, info, healthcheck, debug
```

Comment:
If we're adding this to the existing daemon.loglevel `ini`, are we forcing anyone who wants this functionality to also have error+warning+info? What if they want the healthcheck, but don't want logs any higher than `warning`?

Reply:
I think it would be better suited in the long run to make this feature it's own INI value that can be set independently of other values. In my opinion it serves a specific purpose that doesn't fit neatly into the existing `newrelic.daemon.loglevel` INI, or any other currently existing INI's that I can think of.

Reply:
My understanding is the request was for a log level more verbose than 'info' that provided the health information.  There was no specification of picking and choosing the log level messages that would be reported.